### DISCUSSION TITLE
Display all talents' stats in the Builder

### DISCUSSION THREAD

**Discussion by HenrikRaz** [Heroes-Profile/heroesprofile#994] [Ideas]
**Change request**
When a hero is selected in the talent builder, I would like the stats of all talent's to be visible. 
Also, when a talent on a tier is selected, I would still like to see the other talents' winrate/games from the same tier with the selected build.

On the screenshot here, you could show the winrate of Wild Vigor and Splintered Spear with Leap. And show the winrate of Vines with Choking Pollen. For example. 

<img width="416" alt="Builder" src="https://github.com/Heroes-Profile/heroesprofile/assets/20615795/833d8ceb-0cb3-46c2-b456-5b625e0ce9c6">

**Comment by Zemill** [comment]
I do not believe adding this makes sense as it could be confusing, and to accurately calculate it would add in x#Talents performance hit
